### ISSUE TITLE
Issue with LIST -a /test

### ISSUE THREAD

**Issue by guppy** [giampaolo/pyftpdlib#407]
A client of ours noticed that Netbeans wasn't working properly with our FTP server but worked fine with PureFTPd and vsftpd. Here's a sample of the work flow done through lsftp:

lftp guppy@localhost:~> cd test
cd ok, cwd=/test                 
lftp guppy@localhost:/test> ls -a
-rw-rw-rw-   1 ftpuser  ftpuser      1939 Nov 22 18:18 utils.pm
lftp guppy@localhost:/test> ls -a /test
lftp guppy@localhost:/test>

For the sake of this issue, our Netbeans project is set to use / as it's root. The first thing it does is a LIST -a / (which fails as well, same issue), I put a hack in place to strip "-a /" and now I can get a file listing, if I click on one of the directories that it showed me, the first thing it does is a CWD to that directory and the LIST -a /directory.

This works fine in PureFTPd and vsftpd but not in pyftpdlib. I'm not sure what the standard is but given two very popular FTP servers support it, I think pyftpdlib should as well.

**Comment by giampaolo** [comment]
Yes, this should definitively be handled by pyftpdlib. I don't have time to work on a patch. 
If you want, feel free to submit a pull request.
This is where it is supposed to be done:
https://github.com/giampaolo/pyftpdlib/blob/6165a40fd4f7c7cd2d549dbb816b3208091ffdff/pyftpdlib/handlers.py#L1465
...as you see, the code already handles the case where uncompliant clients specify "-a", "-l" etc. but not the case where a path occurs after "-a" / "-l" (e.g. LIST -l /foo/bar).
